File: app/src/main/java/com/countrydelight/lintruleproject/java_test/variable_scope/RightGlobalVariableScope.java
package com.countrydelight.lintruleproject.java_test.variable_scope;

public class RightGlobalVariableScope {

    private final String rightGlobalVariableScope = "right_scope";

    public void rightGlobalVariableScope1() {
        System.out.println(rightGlobalVariableScope.length());
    }

    private void rightGlobalVariableScope2() {
        System.out.println(rightGlobalVariableScope.length());
    }

}

File: app/src/main/java/com/countrydelight/lintruleproject/kotlin_test/variable_scope/RightGlobalVariableScope.kt
package com.countrydelight.lintruleproject.kotlin_test.variable_scope

class RightGlobalVariableScope {

    private var rightGlobalVariableScope = "right_scope"

    fun rightGlobalVariableScope1() {
        print(rightGlobalVariableScope.length)
    }

    fun rightGlobalVariableScope2() {
        print(rightGlobalVariableScope.length)
    }
}